src/libYARP_serversql/src/yarpserver.cpp

```diff
@@ -205,6 +205,7 @@ yarpserversql_API int yarpserver_main(int argc, char *argv[]) {
 
     if (options.check("help")) {
         printf("Welcome to the YARP name server.\n");
+        printf("  --write                  The configuration file will be written with the address for the name server, if found.\n");
```

Comment:
The length of the line is a bit long compared to the other ones, please split it in 2 lines (see --portdb and --subdb)
Also I'd say "Write IP address and socket on the configuration file" or something similar, for coherence with the other help lines...